dnn/data/converters/roco.py

```diff
@@ -0,0 +1,97 @@
+from absl import app, flags
+from absl.flags import FLAGS
+
+import os
+import xml.etree.ElementTree as xmlTree
+
+import numpy as np
+import tensorflow as tf
+from tqdm import tqdm
+
+from dnn.data.converters.utils import (
+    bytes_feature,
+    CLASS_NAMES,
+)
+from dnn.utils.mem import tf_set_memory_growth
+
+flags.DEFINE_string('input', '../DJI ROCO', 'the path for input ROCO dataset (please unzip by yourself)')
+flags.DEFINE_string('output', '../DJI ROCO TFRecord', 'the path for output TFRecord file(s)')
```

Comment:
ditto

Reply:
I think there is no way to avoid this, unless we put datasets directly in `converters`, which doesn't make sense at all.